src/components/ModuleSidebar/moduleSidebar.scss

```diff
@@ -0,0 +1,46 @@
+$sidebar-links-color: #8f5806;
+
+.d-aside{
+  padding-top: 2rem;
+}
+
+ .d-sidebar aside {
+  padding: 10px;
+}
+
+.d-links li.module-active {
+  background: #523304;
```

Comment:
I think you're reusing this bg color, consider using a variable

Reply:
Okay